CHANGELOG.rst

```diff
@@ -12,6 +12,14 @@ Change Log
 Unreleased
 ----------
 
+[10.1.0] - 2024-01-26
+---------------------
+
+* Added toggle EDX_DRF_EXTENSIONS[ENABLE_JWT_AND_LMS_USER_EMAIL_MATCH]:
+
+    * If enabled, makes sure that the user email in JWT cookies and LMS user email matches
+    * If email matches, it allows authentication otherwise raise JwtUserEmailMismatchError error.
```

Comment:
```suggestion
* Added permanent toggle EDX_DRF_EXTENSIONS[ENABLE_JWT_AND_LMS_USER_EMAIL_MATCH]:

    * This toggle should only get enabled in the LMS, and should remain disabled in all other services.
    * If enabled, makes sure that the user email in JWT cookies and LMS user email matches
    * If email matches, it allows authentication otherwise raise JwtUserEmailMismatchError error.
```